themes/openy_themes/openy_rose/openy_rose.theme

```diff
@@ -266,7 +266,7 @@ function openy_rose_page_attachments_alter(array &$page) {
  */
 function openy_rose_preprocess_page(array &$variables) {
   // Set custom theme variables for page when they are set.
-  $image_fields = theme_get_setting('oepny_rose_image_fields');
+  $image_fields = \Drupal::config('openy_rose.settings')->get('oepny_rose_image_fields');
```

Comment:
I think there's a typo in the config name `oepny_rose_image_fields` should be `openy_rose_image_fields`.
1. This should be fixed in this function.
2. This should be fixed here as well https://github.com/ymcatwincities/openy/blob/5a2478bbeb896ca2a3abe9ac88daa76c4baf8a19/themes/openy_themes/openy_rose/config/install/openy_rose.settings.yml#L2.
3. Upgrade path should be provided:
- Get value of `oepny_rose_image_fields`
- Save it as `openy_rose_image_fields`
- Remove `oepny_rose_image_fields` property.